nntrainer/tensor/tensor.h

```diff
@@ -675,19 +675,23 @@ class Tensor {
                                   float beta = 0.0f) const;
 
   /**
-   * @brief     Transpose Tensor
-   * @param[in] direction to transpose ex) 0:2:1
-   * @retval    Calculated Tensor
+   * @brief Transpose Tensor
+   *
+   * @param direction to transpose ex) 0:2:1
+   * @param is_FC if layer == FC only transpose shape and clone data
+   * @return Tensor
    */
-  Tensor transpose(const std::string &direction) const;
+  Tensor transpose(const std::string &direction, bool is_FC = false) const;
 
   /**
    * @brief Transpose Tensor
    * @param direction to transpose ex) 0:2:1
    * @param[out] Tensor to save to, dimension is always reshaped.
+   * @param is_FC if layer == FC only transpose shape and clone data
```

Comment:
It's strange. I think transpose operation cannnot distinguish layer....
If you needs this, add option like `shape_only`, and then copyData from it in outer code would be right.